core/heap.c

```diff
@@ -3335,6 +3341,7 @@ heap_vmareas_synch_units()
 {
     heap_unit_t *u, *next;
     /* make sure to add guard page on each side, as well */
+    // NOCHECK
```

Comment:
What does this NOCHECK comment signify?

Reply:
Ah, I missed that, thank you.  That function needs a logic update for the new guard rules.  Fixed and removed the marker comment.